src/types/product-document.ts

```diff
@@ -0,0 +1,16 @@
+import {Document} from 'mongoose';
+
+export interface ProductDocument extends Document {
+    _id: string,
+    description: string,
+    price: number,
+    quantity: number,
+    imageUrl: string,
+    toClient: () => {
```

Comment:
why the duplication in all fields? work with types

Reply:
done. is that what you meant?

Reply:
yes